src/components/Board.js

```diff
@@ -8,26 +8,123 @@ import NewCardForm from './NewCardForm';
 import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    this.emojiLibrary = require('emoji-dictionary');
+
 
     this.state = {
       cards: [],
     };
   }
 
+  emojify = (emojiTextName) => {
+    let validEmojiCheck = this.emojiLibrary.names.includes(emojiTextName)
+    return validEmojiCheck ? this.emojiLibrary.getUnicode(emojiTextName) : emojiTextName
+  }
+
+  componentDidMount() {
+    const GET_CARDS_URL = 'https://inspiration-board.herokuapp.com/boards/mightymichelle/cards';
```

Comment:
it would be nice if you used the `prop`s passed into `Board` from `App` -- `url` and `boardName`!